### ISSUE TITLE
MEI_4.0 example problems and observations on the way to MEI 5.0

### ISSUE THREAD

**Issue by bwbohl** [music-encoding/sample-encodings#77]
Checking the current MEI_4.0 examples when evaluating the conversion to MEI 5.0 I stumbled over the following encoding problems:

* many files have the mei30to40.xsl listed in `<appInfo>` but do not have a corresponding `<change>` pointing to the application's `@xml:id` similar to the one found in other files, e.g.:
  ```xml
   <change n="1" resp="#app_20193104017386">
      <changeDesc>
         <p>Converted to MEI version 4.0.0 using mei30To40.xsl, version 1.0 beta</p>
      </changeDesc>
      <date isodate="2019-01-03"/>
    </change>
  ```
  
* in other cases, it might be vice-versa
* the files that do not contain an `<meiHead>` consequently have none of this documentation which seems a bit inconsistent
* in MEI_4.0 there are
   |#|Description|
   |   ---:| ------------- |
   |253 |files in total spread across the subdirectories|
   |75    |are .pdf|
   |145 | are .mei|
   |33| are .xml|
   
   While the question of whether to clone the existing PDFs might be a separate issue, the extensions are used inconsistently. Should we rename everything to `.mei`?
* The referenced schema in all files is `http://music-encoding.org/schema/4.0.0/mei-all.rng`
** This probably should be updated to `https` and to `4.0.1`?
** Moreover, it does not really apply to all files since some of them should be validated with the `anyStart` schema or even with the schemata from other customisations.

Although I don't think we urgently need fixed MEI 4 examples once MEI 5 is out, some of these issues will also throw errors or warnings in the updated MEI 5 examples.

Happy to get everybody's input on this, especially from the @music-encoding/maintainers.

**Comment by ahankinson** [comment]
IMO, we should leave `MEI_4.0` as-is and not touch it for the upgrade. If we want to revisit it later we can.

However, if you want to make things more consistent with 5.0, then I agree that the `.xml` files should become `.mei`, and the appropriate schema attached to the 5.0 files. 

> some of these issues will also throw errors or warnings in the updated MEI 5 examples.

which issues cause errors or warnings? can we get around them?

**Comment by bwbohl** [comment]
it's only warnings: especiallly those about `change/@resp` pointing to a non existing xml:id. (encodingDesc/appInfo/application missing). All others are <respStmt> without a resp or `@role`s. Which might be ignorable.

**Comment by bwbohl** [comment]
I'm not for updating MEI 4 files unless it brings better (more valid) results for the MEI 5 files; e.g. fixing the resp:s in change or respStmt in MEI 4 would make more sense as it would then also be fixed in MEI 5.

**Comment by ahankinson** [comment]
OK... what sort of additional feedback do you need to help you out with music-encoding/encoding-tools#30?

**Comment by musicEnfanthen** [comment]
Just for the record: Cf. #52

**Comment by bwbohl** [comment]
Thx for the feedback these issues will be resolved in the upcoming sample-encoding v4.01 release https://github.com/music-encoding/sample-encodings/tree/release/v4.0.1
